packages/bbui/src/Form/DatePicker.svelte

```diff
@@ -13,8 +13,9 @@
 
   const dispatch = createEventDispatcher()
   const onChange = e => {
-    value = e.detail
-    dispatch("change", e.detail)
+    const isoString = e.detail.toISOString()
```

Comment:
Was this the cause of the bad date sorting issue - that it was storing actual JS objects sometimes and strings other times? I know Flatpickr emits JS date objects (and I hate Flatpickr) but there could be consequences of changing this somewhere that might have been written to expect the date objects, so it just might need checked well.

Reply:
Yep, there was a mismatch between the way dates are stored in the system, between handlebars representation (which is ISO) and the way flatpickr stores it, which is just `new Date().toString()` from the looks of it. Updating this still seems to work with flatpickr and store the right format in the DB.

Reply:
Cool, all good then.